### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by luisramalho**
Portuguese translation for the README.md

### CODE REVIEW

File: translations/Portuguese/README.pt.md [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+**Bom**
+```xml
+<string name="error.message.network">Network error</string>
+<string name="error.message.call">Call failed</string>
+<string name="error.message.map">Map loading failed</string>
+```
+
+Não escreva os valores do strings em letras maiúsculas. Mantenha-se dentro das convenções normais de texto (exemplo: coloque o primeiro caráter em letras maiúscula). Se precisa de mostrar o string em letras maiúsculas, então use por exemplo o atributo [`textAllCaps`](http://developer.android.com/reference/android/widget/TextView.html#attr_android:textAllCaps) na própria TextView.
+
+**Bad**
```
Comment (staltz) [inline translations/Portuguese/README.pt.md]: This should be translated too

File: translations/Portuguese/README.pt.md [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+```xml
+<string name="error.message.call">CALL FAILED</string>
+```
+
+**Good**
```
Comment (staltz) [inline translations/Portuguese/README.pt.md]: This should be translated too

File: translations/Portuguese/README.pt.md [hunk 3]
```diff
@@ -0,0 +1,562 @@
...
+</resources>
+```
+
+Nas margens e espaçamentos deverá usar as dimensões `spacing_****` para o layout, ao invés de colocar valores fixos, da mesma forma que o strings são tratados. Isto irá dar um look-and-feel consistente, ao mesmo tempo que irá fazer com que seja mais fácil organizar e modificar estilos e layouts.
+
+**strings.xml**
+
+Dê o nome aos strings com keys que fazem lembrar namespaces, e não tenha receio de repetir um valor para duas ou mais keys. Linguagens são complexas, portanto namespaces são necessários para trazer contexto e quebrar ambiguidade.
+
+**Mau**
```
Comment (staltz) [inline translations/Portuguese/README.pt.md]: Personally, I'd say "Ruim". In pt-br "Mau" has connotation as "Evil", while it should be just "Bad".

Comment (luisramalho) [inline translations/Portuguese/README.pt.md]: "Ruim" is not really a word we use in pt-pt but understandable. I'd change the Good/Bad to "Correto/Incorreto", but that changes a bit the meaning making it more assertive.

**Pull request merged**